unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioSource.cs

```diff
@@ -162,11 +164,12 @@ private void ApplyLoadedAudioClip(DCLAudioClip clip)
             }
 
             Model model = (Model) this.model;
-            if (audioSource.enabled && model.playing && !audioSource.isPlaying)
+            if (audioSource.enabled && model.playing && playedAtTimestamp != model.playedAtTimestamp)
             {
                 //To remove a pesky and quite unlikely warning when the audiosource is out of scenebounds
                 audioSource.Play();
             }
```

Comment:
It looks like (I didn't test it) if I do the following:
```
audio.playOnce()
audio.playing = false
audio.playing = true
```
the audio won't play any more unless `playOnce` is called called again?

Reply:
I reread what Brian said, and he thought about this case. I force-pushed the change.
What do you think now?